stp_core/common/log.py

```diff
@@ -2,42 +2,36 @@
 import logging
 import os
 import sys
-
 from ioflo.base.consoling import getConsole, Console
 from stp_core.common.logging.TimeAndSizeRotatingFileHandler import TimeAndSizeRotatingFileHandler
 from stp_core.common.util import Singleton
-from stp_core.util import getConfig
+from stp_core.common.logging.handlers import CliHandler
 
 TRACE_LOG_LEVEL = 5
 DISPLAY_LOG_LEVEL = 25
 
-# TODO: move it to plenum-util repo
+# TODO: move it to plenum-utils
 
 
-def getlogger(name: object = None) -> object:
-    return Logger().getlogger(name)
+class CustomAdapter(logging.LoggerAdapter):
+    def trace(self, msg, *args, **kwargs):
+        self.log(TRACE_LOG_LEVEL, msg, *args, **kwargs)
 
+    def display(self, msg, *args, **kwargs):
+        self.log(DISPLAY_LOG_LEVEL, msg, *args, **kwargs)
 
-class TestingHandler(logging.Handler):
-    def __init__(self, tester):
-        """
-        Initialize the handler.
-        """
-        super().__init__()
-        self.tester = tester
 
-    def emit(self, record):
-        """
-        Captures a record.
-        """
-        self.tester(record)
+def getlogger(name: object = None) -> object:
+    return Logger().getlogger(name)
 
 
 class Logger(metaclass=Singleton):
     def __init__(self, config=None):
+        from plenum.common.config_util import getConfig
```

Comment:
Also, somehow it feels like stp repo is becoming collection of utility or common code. As most of the other repos will depend on stp_core, so it may not look like big concern right now, but somehow I was thinking to have a util (or common) repo which can be used by any other repo (whether in evernym or sovrin-foundation), what do you think?